demo/admin/src/App.tsx

```diff
@@ -147,7 +147,10 @@ class App extends React.Component {
                                                                                         render={({ match: { params } }) => {
                                                                                             const category = urlParamToCategory(params.category);
 
-                                                                                            if (category === undefined) {
+                                                                                            if (
+                                                                                                !category ||
+                                                                                                !categories.find((c) => c.category === category)
```

Comment:
You may use `categories.includes(category)` here

Reply:
I can't because categories has the form `Array<{ category: string; label: React.ReactNode }>`

Reply:
could be changed to `Record<string, { label: React.ReactNode }>`?

Reply:
We define this in cms-admin

https://github.com/vivid-planet/comet/blob/99261a7ef4f45a039657d8a2ada2a43fa0679aac/packages/admin/cms-admin/src/pages/pageTree/PageTreeContext.ts#L8

Changing this would be a breaking change